tests/Unit/Domain/Test_Orientations.cpp

```diff
@@ -0,0 +1,181 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
```

Comment:
If I can copy it I'd like to know if two things are equal. Is there any reason to not be able to compare these while being able to copy?